app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1425,10 +1425,15 @@ impl GraphEditorModelWithNetwork {
             AddNodeEvent => default(),
             StartCreationEvent | ClickingButton if selection.is_some() =>
                 self.find_free_place_under(selection.unwrap()),
-            StartCreationEvent => mouse_position,
+            StartCreationEvent =>
+                self.new_node_position_at_mouse_aligned_to_close_nodes(mouse_position),
             ClickingButton =>
                 self.find_free_place_for_node(screen_center, Vector2(0.0, -1.0)).unwrap(),
-            DroppingEdge { .. } => mouse_position,
+            DroppingEdge { edge_id } => self
+                .new_node_position_at_mouse_aligned_if_close_to_edge_source_node(
```

Comment:
1. The variables `new_node_position_at_mouse_aligned_to_close_nodes` and `new_node_position_at_mouse_aligned_if_close_to_edge_source_node` are inconsistent – one contains the word "if" while the second does not, why?

2. "close to edge source node" is not a valid English phrase and it makes it really hard to understand what this is about.

2. What more information does this name carries to the user than `new_node_position_at_mouse_aligned_if_close_to_source_node`?

Reply:
***Re: 1.***

In the previous review, there was a suggestion to use a name `new_node_position_aligned_if_close_to_node` for the function called by `new_node_position_at_mouse_aligned_to_close_nodes`. Thus, spelled out more precisely, the latter function could be named `new_node_position_at_mouse_aligned_if_close_to_closest_node`. However, in the same review the name `new_node_position_at_mouse_aligned_to_close_nodes` was also specifically suggested, skipping the `if_close...` fragment. Based on the comment above and the comments from the previous review, I now understand that the suggestion here is to drop the `if_close_` fragment from the `new_node_position_at_mouse_aligned_if_close_to_source_node` name (thus making it `new_node_position_at_mouse_aligned_to_source_node`), however to still keep it in the `new_node_position_aligned_if_close_to_node` name (as this name was specifically suggested in the previous review) - so I'm changing the code that way. Or is the suggestion now changed to also drop it from the `new_node_position_aligned_if_close_to_node` name for consistency (and thus make it `new_node_position_aligned_to_node`)?